init/common.js

```diff
@@ -61,6 +61,8 @@ const bootstrapDatabase = async () => {
   // creating project without associating team
   await createProject(interestId, userId, config.projectWithoutTeam, false);
   console.log("Project created without associated team");
+  await createOpportunities(interestId);
+  console.log("Opportunities created");
```

Comment:
Hey @nikita-pardeshi-github I realize that here we are using plural. In this case we are creatting opportunities or just one opportunity right?

Reply:
@tiagopazhs we are creating more than one opportunities at this instance as mentioned in user config. Considering opportunities declared in user config we can keep it plural for now, would that be okay? Additionally, can you please approve if you do find it good to merge now?